extensions/amp-iframe/amp-iframe-vs-native-iframe.md

```diff
@@ -0,0 +1,22 @@
+# amp-iframe vs native iframe
+
+## Background
+
+amp-iframe has been an AMP component since ~2015. It was designed to make native iframes more secure while
+enforcing restrictions like its position outside the top 75% or 600px in an AMP document to not degrade user experience. With the migration of amp-iframe to Bento, we want to assess if we retire amp-iframe component in favor of native iframes, which now support new features like lazy loading.
```

Comment:
Nit: extra carriage return?

Also, there are multiple options here. Could we allow `iframe` in addition to `amp-iframe`?